daofund-score/src/main/java/com/iconbet/score/daofund/DaoFund.java

```diff
@@ -0,0 +1,303 @@
+    private static final BigInteger X_6 = new BigInteger("1000000"); // 10 ** 6
+    private static final BigInteger BATCH_SIZE = BigInteger.valueOf(100L);
+    
+
+	private final ArrayDB<Address> admins = Context.newArrayDB(ADMINS, Address.class);
+	private final VarDB<BigInteger> withdraw_count = Context.newVarDB(WITHDRAW_COUNT, BigInteger.class);
+	private final DictDB<String,LinkedList> withdraw_record = Context.newDictDB(WITHDRAW_RECORD, LinkedList.class);
+
+	
+	/***
+	  def on_update(self) -> None:
+	        super().on_update()
+	        self.admins.put(self.owner)
+	        self.withdraw_count.set(0)
+	 * @param <T>
+	  ***/
+	
+	
+	private <T> boolean remove_array_item(T value, ArrayDB<T> arraydb ) {
+		
+		return Boolean.FALSE;
+		
+	}
+		
+	
+	/***
+	
+	def remove_array_item(array_db, target) -> bool:
+	    _out = array_db[-1]
+	    if _out == target:
+	        array_db.pop()
+	        return True
+	    for index in range(len(array_db) - 1):
+	        if array_db[index] == target:
+	            array_db[index] = _out
+	            array_db.pop()
+	            return True
+	    return False
+***/
+
+	@External(readonly = true)
+	public String name() {
+		/***
+		:return: name of the Score
+		***/
+		
+		return TAG;
+	}
+	
+	@External
+	public void add_admin(Address _admin) {
+		Address sender = Context.getCaller();
+		Address owner = Context.getOrigin();
+		if (!sender.equals(owner)) {
+			Context.revert(TAG + ": Only admins can set new admins.");
+		}
+		
+		if (!containsInArrayDb(_admin, this.admins)) {
+			this.admins.add(_admin);
+			AdminAdded(_admin);
+			
+		}else {
+			Context.revert(TAG + ":  "+ _admin +" is already on admin list.");
+		}
+	}
+	
+	@External
+	public void remove_admin(Address _admin) {
+		Address sender = Context.getCaller();
+		Address owner = Context.getOrigin();
+		if (!sender.equals(owner)) {
+			Context.revert(TAG + ": Only admins can remove admins.");
+		}
+		
+		if (_admin.equals(owner)) {
+			Context.revert(TAG + ": Owner address cannot be removed from the admins list.");
+		}
+		
+		if (containsInArrayDb(_admin, this.admins)) {
+			remove_array_item(_admin, this.admins);
+			AdminRemoved(_admin);
+		}else {
+			Context.revert(TAG + ":  "+ _admin +" not in Admins List");
+		}
+	}
+	
+	@External(readonly = true)
+	public List<Address> get_admins() {
+		
+		List<Address> addressList = new ArrayList<>();
+
+		for (int i=0; i< this.admins.size(); i++) {
+			addressList.add(this.admins.get(i));
+		}
+		return addressList;
+	}
+	
+	/***
+	 * Add fund to the daoFund wallet
+	 ***/
+	@External
+	@Payable
+	public void add_fund() {
+		
+	}
+		 
+	@External
+	public void withdraw_fund( Address _address, BigInteger _amount, String _memo) {
+		Address sender = Context.getCaller();
+		
+		if (!containsInArrayDb(sender, this.admins)) {
+			Context.revert(TAG + ": Only admins can run this method.");
+		}
+		//  _available_amount = self.icx.get_balance(self.address)
```

Comment:
i do not see Context.getBalance java call

Reply:
done.